### DISCUSSION TITLE
How to make the model perfectly match geojson?

### DISCUSSION THREAD

**Discussion by pengguixin** [visgl/deck.gl#8207] [Q&A]
【I don't know English, the following is machine translation. If there is any disrespect, please forgive me】
According to a set of GeoJSON files, I imported them into Blender to create a model. Then I imported the model and GeoJSON into deck.gl, but found that the model does not match the GeoJSON. It is known that the center point of the model is correct. Additionally, I exported the model from Blender as an SHP file and converted it to GeoJSON, which perfectly matched the original GeoJSON. Therefore, the scale of the model should be correct. However, the model does not match the GeoJSON. I hope to receive assistance and would greatly appreciate it.
<img width="457" alt="geo-glb" src="https://github.com/visgl/deck.gl/assets/148191846/8587e048-d2bd-40d7-bbdb-1337aec90c2a">

github: https://github.com/pengguixin/deck-geo-glb

**Comment by Pessimistress** [comment]
https://github.com/visgl/deck.gl/discussions/8085#discussioncomment-6864667
